### PR TITLE
Homework_7_Java_Arrays_Loops

### PR DESCRIPTION
**Pull request by olyanemyrovska**

### CODE REVIEW

File: Homework_7_Java_Operators_Conditions/JavaArraysAndLoops/src/StrongestRobot.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+public class StrongestRobot {
+    public static void main(String[] args) {
+        int[] weights = {300, 450, 200, 500, 350, 900, 0};
+
+        int strongestRobot = -1;
```
Comment (rodikno) [inline Homework_7_Java_Operators_Conditions/JavaArraysAndLoops/src/StrongestRobot.java]: not a mistake, but just wondering - why initializing to `-1`?

File: Homework_7_Java_Operators_Conditions/JavaArraysAndLoops/src/StrongestRobot.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        int maxWeight = 0;//assuming that min weight is 0
+
+        for (int i = 0; i < weights.length; i++) {
+            if (weights[i] > maxWeight) {
+                maxWeight = weights[i];
+                strongestRobot = i;
+            }
+        }
+        System.out.println("Strongest robot number is: " + strongestRobot);
```
Comment (rodikno) [inline Homework_7_Java_Operators_Conditions/JavaArraysAndLoops/src/StrongestRobot.java]: `Переконайтесь, що вивели в консоль ім*я найсильнішого робота **і вагу, яку він підняв**` :)
але зараховано

File: Homework_7_Java_Operators_Conditions/JavaArraysAndLoops/src/WaterIntakeTracker.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        int sum = 0;
+        while (waterIntake.length > i) {
+            if (waterIntake[i] >= 2) {
+                sum += waterIntake[i];
+            }
+            i++;
+        }
+        System.out.println("You have taken " + sum + " cups of water during this week.");
+
+//Option 2 with 'continue'
```
Comment (rodikno) [inline Homework_7_Java_Operators_Conditions/JavaArraysAndLoops/src/WaterIntakeTracker.java:16]: Навіть два варіанти! Чудова робота, прийнято 👍

**Review by rodikno — APPROVED** [review]

File: Homework_7_Java_Operators_Conditions/JavaArraysAndLoops/src/StrongestRobot.java (hunk 1 above)
Comment (olyanemyrovska) [reply Homework_7_Java_Operators_Conditions/JavaArraysAndLoops/src/StrongestRobot.java]: Initially, I wanted to set the index to 0, but then I considered what would happen if the array were empty or all values were the same. In such cases, returning 0 would not be correct. So, I googled for a solution, and the recommendation was to use -1.
By the way, which option is more appropriate here?

File: Homework_7_Java_Operators_Conditions/JavaArraysAndLoops/src/StrongestRobot.java (hunk 2 above)
Comment (olyanemyrovska) [reply Homework_7_Java_Operators_Conditions/JavaArraysAndLoops/src/StrongestRobot.java]: Пропустила, дякую за уважність, поправлю)

**Pull request merged**